src/app/public/src/modules/code-block/code-block.component.html

```diff
@@ -1,6 +1,16 @@
 <div class="stache-code-block">
-  <div #codeFromContent class="stache-code-input"><ng-content></ng-content></div>
-  <pre class="stache-code-output"><code
-    [ngClass]="getClassName()"
-    [innerHTML]="output"></code></pre>
-</div>
\ No newline at end of file
+  <div #codeFromContent class="stache-code-input">
+    <ng-content></ng-content>
+  </div>
+  <div stacheClipboard clipboardTrigger="copyCodeButton">
+    <pre class="stache-code-output"><code [ngClass]="getClassName()" [innerHTML]="output"></code></pre>
+    <button
+      aria-label="copy to clipboard"
+      class="sky-btn sky-btn-default stache-clipboard-btn"
+      id="copyCodeButton"
```

Comment:
Avoid using `id` since this could be duplicated several times on a page. You can target elements using Angular's Template Reference Variables.

```
<div stacheClipboard clipboardTrigger="copyCodeButton">
...
<button #copyCodeButton>
```